pkgs/tools/misc/ffsend/default.nix

```diff
@@ -29,7 +29,7 @@ buildRustPackage rec {
 
   nativeBuildInputs = [ cmake pkg-config installShellFiles ];
   buildInputs =
-    if stdenv.isDarwin then (with darwin.apple_sdk.frameworks; [ CoreFoundation CoreServices Security AppKit ])
+    if stdenv.isDarwin then [libiconv] ++ (with darwin.apple_sdk.frameworks; [ CoreFoundation CoreServices Security AppKit ])
```

Comment:
While we are at it: can we remove darwin from inputs?

Reply:
done

Reply:
what do you mean by remove darwin?